src/librustc/ty/mod.rs

```diff
@@ -1250,6 +1332,17 @@ pub struct ParamEnv<'tcx> {
     /// want `Reveal::All` -- note that this is always paired with an
     /// empty environment. To get that, use `ParamEnv::reveal()`.
     pub reveal: traits::Reveal,
+
```

Comment:
I find `ParamEnv` containing a `universe` rather unsatisfying - it doesn't fit in my mental model.